setup.py

```diff
@@ -12,13 +12,10 @@
 with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
-requirements = [
-    "Click>=6.0",
-    "matchms==0.8.2",
-    "pandas==1.1.3",
-    "requests==2.24.0",
-    "confuse==1.4.0",
-]
+with open("requirements.txt") as requirements_file:
+    requirements = requirements_file.read().split("\n")
+    if "" in requirements:
```

Comment:
Added this line, because the requirements.txt can contain a paragraph after the last package.

Reply:
Ah, this was causing the problem?